imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -608,7 +632,7 @@ private File localCdrDir() {
     @Override
     public void verifyDetailFileChecksumAndCount(CdrFileNotificationRequest request) {
 
-        LOGGER.debug("Starting CDR Process Phase 1");
+        LOGGER.info("Phase 1 - Start");
```

Comment:
Any reason why you removed "CDR Process"?  Seems like a useful string to search for